### ISSUE TITLE
Error: DEM processing 'mode' not recognized when using color-relief

### ISSUE THREAD

**Issue by wiesehahn** [USDAForestService/gdalraster#560]
I am not entirely sure if I got something wrong here but it seems mode `color-relief` does not work here.
``` r
elev_file <- system.file("extdata/storml_elev.tif", package="gdalraster")
out_file <- file.path(tempdir(), "storml_out.tif")

rgb <- data.frame(c(2500, 2750, 3000), c(112, 180, 245), c(147, 192, 245), c(141, 180, 245))
txt <- fs::file_temp(ext = "txt")
write.table(rgb, txt, sep = "\t", row.names = FALSE, col.names = FALSE)


gdalraster::dem_proc("color-relief", elev_file, out_file, color_file = txt)
#> Error: DEM processing 'mode' not recognized
```

<sup>Created on 2024-12-07 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

**Comment by ctoney** [comment]
That was indeed a bug, reported in #428 and fixed in #430. We have not done a release on CRAN since that fix though. A workaround would be to use "color_relief" with an underscore instead of "color-relief". Another option could be to install the development version from GitHub or R-Universe. We're probably a couple months out from the next CRAN release which will add the GDAL vector bindings, although it's possible we could backport a few small bug fixes and do a maintenance release sooner than that.

**Comment by ctoney** [comment]
Edit: sorry, I just realized the underscore is not actually a workaround. It was recognized that way by the function but still not passed through correctly to GDAL. Is the development version an option for you in the short term?

**Comment by wiesehahn** [comment]
Yes sure, no worries. Should have checked that anyway before reporting fixed bugs ;)
